CHANGELOG.md

```diff
@@ -4,6 +4,52 @@
 
 <!-- next version -->
 
+## 1.0.0-rc5/0.71.0
```

Comment:
Format is different from log entry below (this one does not have a `v`). Is this a concern?

Reply:
Yup, saw that too. See https://github.com/open-telemetry/opentelemetry-collector/pull/7141